src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Book not found")
+public class BookNotFoundException extends RuntimeException{
+
+    public BookNotFoundException() {
```

Comment:
If I'm not mistaken, you should call the super() method to handle the construction of the RuntimeException.